packages/core/admin/admin/src/content-manager/components/PreviewWysiwyg/utils/mdRenderer.js

```diff
@@ -12,6 +11,12 @@ import mark from 'markdown-it-mark';
 import sub from 'markdown-it-sub';
 import sup from 'markdown-it-sup';
 
+const loadCss = async () => {
```

Comment:
Same goes here, I personnally wouldn't load CSS in a lazy chunk for now (DS is coming)

Reply:
The DS won't solve this issue here I am loading the lib required in order to have to have a custom preview of the markdown.